Tests/XCUITests/JumpBackInTests.swift

```diff
@@ -0,0 +1,175 @@
+
+        // Visit YouTube in private browsing
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        navigator.openURL("https://www.youtube.com")
+        waitUntilPageLoad()
+
+        // Open a new tab in normal browsing and check the "Jump Back In" section.
+        navigator.toggleOff(userState.isPrivate, withAction: Action.ToggleRegularMode)
+        navigator.goto(NewTabScreen)
+        closeKeyboard()
+
+        // Twitter should be in "Jump Back In".
+        waitForExistence(app.staticTexts["Jump Back In"])
+        waitForExistence(app.cells["JumpBackInCell"])
+        waitForExistence(app.cells["JumpBackInCell"].staticTexts["Twitter"])
+        waitForNoExistence(app.cells["JumpBackInCell"].staticTexts["YouTube"])
+
+        // Visit "amazon.com" and check the "Jump Back In" section.
+        navigator.openURL("https://www.amazon.com")
+        waitUntilPageLoad()
+
+        navigator.goto(TabTray)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        closeKeyboard()
+
+        // Amazon and Tiwtter are visible in the "Jump Back In" section
+        waitForExistence(app.staticTexts["Jump Back In"])
+        waitForExistence(app.cells["JumpBackInCell"])
+        waitForExistence(app.cells["JumpBackInCell"].staticTexts["Amazon"])
+        waitForExistence(app.cells["JumpBackInCell"].staticTexts["Twitter"])
+        waitForNoExistence(app.cells["JumpBackInCell"].staticTexts["YouTube"])
+
+        // Tap on Twitter from "Jump Back In"
+        app.cells["JumpBackInCell"].staticTexts["Twitter"].tap()
+
+        // The view is switched to the twitter tab
+        var url = app.textFields["url"].value as! String
+        XCTAssertEqual(url, "twitter.com/")
+
+        // Open a new tab in normal browsing
+        navigator.goto(TabTray)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        closeKeyboard()
+
+        // Check the "Jump Back In Section".
+        waitForExistence(app.staticTexts["Jump Back In"])
+        waitForExistence(app.cells["JumpBackInCell"])
+
+        // Amazon is visible in "Jump Back In".
+        waitForExistence(app.cells["JumpBackInCell"].staticTexts["Amazon"])
+
+        // Close the amazon tab.
+        navigator.goto(TabTray)
+        if isTablet {
+            waitForExistence(app.navigationBars.segmentedControls["navBarTabTray"])
+        } else {
+            waitForExistence(app.navigationBars.staticTexts["Open Tabs"])
+        }
+        app.cells["Amazon.com. Spend less. Smile more."].buttons["tab close"].tap()
+
+        // Revisit the "Jump Back In" section
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        closeKeyboard()
+
+        // The "Jump Back In" section is still here with twitter listed
+        waitForExistence(app.staticTexts["Jump Back In"])
+        waitForExistence(app.cells["JumpBackInCell"])
+        // FXIOS-5448 - Amazon should not be listed because we've closed the Amazon tab
```

Comment:
Workaround: Open an existing "Homepage" tab or open another new tab. There "Jump Back In" section is refreshed.